### ISSUE TITLE
Reproducing results from the paper

### ISSUE THREAD

**Issue by Michael-T-McCann** [camlab-ethz/poseidon#16]
I'd like to reproduce the results in the Poseidon paper, ideally https://arxiv.org/pdf/2405.19101 Table 1. I have downloaded a dataset and a pretrained model. I successfully ran
```
CUDA_VISIBLE_DEVICES=0 python -m scOT.inference --data_path ${PDEGYM_ROOT}/CE-RM --dataset fluids.compressible.RichtmyerMeshkov --model_path ~/.cache/huggingface/hub/models--camlab-ethz--Poseidon-B/snapshots/702667ed6b82ece5021cc16b646feec2bad32d70/ --file ~/output.csv --ckpt_dir none
```
I would like guidance on how to map the results in `~/output.csv` to Table 1. Am I running `scOT.inference` with the correct arguments to make the results comparable to those in the table? Which (if any) column(s) in `~/output.csv` should I expect to match with values in the table?

**Comment by maximilianherde** [comment]
First of all, Table 1 of the paper lists efficiency and accuracy gains which are not given in the output file. The output file only stores raw errors. The paper describes how to retrieve these gains from the errors (and also from which errors).
Further, we are always finetuning from the pretrained checkpoint.
So, the pipeline to get Table 1 is: finetune on every dataset from a pretrained model, train an FNO from scratch, then do inference on both models and calculate the gains between the finetuned model and FNO.

**Comment by armbrusl** [comment]
Thank you for the reply. Regarding the phrase "finetune on every dataset from a pretrained model," is it possible for you to share the .yaml configuration files (and commands) used for a PDE or multiple PDEs? We are also trying to replicate the results from the paper.
